### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by krmaxwell** [kbandla/APTnotes#98]
I rendered README.md to HTML and ran htmlproof:

```
kmaxwell@newton:~/Data/APTnotes(master)$ htmlproof README.html 
Running ["Images", "Links", "Scripts"] checks on README.html on *.html... 


Checking 236 external links...
Ran on 1 files!


- README.html
  *  internally linking to cisak.perpika.kr/wp-content/uploads/2013/07/2013-08.pdf, which does not exist
  *  internally linking to www.fireeye.com/resources/pdfs/apt28.pdf, which does not exist
  *  External link http://www.malware.lu/Pro/RAP002_APT1_Technical_backstage.1.0.pdf failed: 404 No error
  *  External link http://www.triumfant.com/pdfs/Case_Study_Operation_Aurora_V11.pdf failed: 404 No error
  *  External link http://www.mfa.gov.ge/files/556_10535_798405_Annex87_CyberAttacks.pdf failed: 404 No error
  *  External link http://www.totaldefense.com/Core/DownloadDoc.aspx?documentID=1052 failed: 404 No error
  *  External link https://media.blackhat.com/us-13/US-13-Yarochkin-In-Depth-Analysis-of-Escalated-APT-Attacks-Slides.pdf failed: 526 No error
  *  External link http://download01.norman.no/whitepapers/Cyberattack_against_Israeli_and_Palestinian_targets.pdf failed: 404 No error
  *  External link https://albertmeyerblog.files.wordpress.com/3/5/1/4/7/282874-274153/Aurora_HBGARY_DRAFT.pdf failed: 404 No error
  *  External link https://www.bdcol.ee/files/files/documents/Research/BSDR2009/1_%20Ashmore%20-%20Impact%20of%20Alleged%20Russian%20Cyber%20Attacks%20.pdf failed: 0 SSL peer certificate or SSH remote key was not OK
  *  External link http://www.novetta.com/files/1714/1329/6231/Hikit_Preliminary_Analysis.pdf failed: 404 No error
  *  External link http://www.novetta.com/files/9114/1329/6233/ZoxPNG_Preliminary_Analysis.pdf failed: 404 No error
  *  External link http://www.novetta.com/files/3014/1329/6230/Derusbi_Server_Preliminary_Analysis.pdf failed: 404 No error
  *  External link http://www.crowdstrike.com/sites/default/files/AdversaryIntelligenceReport_DeepPanda_0.pdf failed: 404 No error
  *  External link http://www.mcafee.com/us/local_content/reports/how_can_u_tell.pdf failed: 404 No error
  *  External link http://www.fireeye.com/blog/technical/threat-intelligence/2013/09/new-fireeye-report-world-war-c.html failed: 404 No error
htmlproof 1.5.4 | Error:  HTML-Proofer found 16 failures!
```

**Comment by krmaxwell** [comment]
Current status:

---
kmaxwell@newton:~/Data/APTnotes(link-fixing)$ htmlproof --only_4xx README.html
Running ["Images", "Links", "Scripts"] checks on README.html on *.html... 


Checking 247 external links...
Ran on 1 files!


- README.html
  *  External link http://download01.norman.no/whitepapers/Cyberattack_against_Israeli_and_Palestinian_targets.pdf failed: 404 No error
  *  External link http://www.crowdstrike.com/sites/default/files/AdversaryIntelligenceReport_DeepPanda_0.pdf failed: 404 No error
  *  External link http://www.crowdstrike.com/sites/default/files/AdversaryIntelligenceReport_DeepPanda_0.pdf failed: 404 No error
  *  External link http://www.mfa.gov.ge/files/556_10535_798405_Annex87_CyberAttacks.pdf failed: 404 No error
  *  External link http://www.triumfant.com/pdfs/Case_Study_Operation_Aurora_V11.pdf failed: 404 No error
  *  External link https://albertmeyerblog.files.wordpress.com/3/5/1/4/7/282874-274153/Aurora_HBGARY_DRAFT.pdf failed: 404 No error
  *  External link http://www.totaldefense.com/Core/DownloadDoc.aspx?documentID=1052 failed: 404 No error

htmlproof 1.5.4 | Error:  HTML-Proofer found 7 failures!

**Comment by krmaxwell** [comment]
fixed in e3c9250fd430f4f4e4ec45876111dc66f3f6798d
